css/styles.css

```diff
@@ -0,0 +1,299 @@
+HTML{
```

Comment:
Please, use lowercase for selectors. Both, for tags and for classes/identifiers.